src/api/mapping.ts

```diff
@@ -15,7 +20,40 @@ mappingRouter.post('/', (req, res) => {
   }
   domainKeys.push(mappingObject)
   setData('mappings', domainKeys)
-  res.json(mappingObject)
+  const projectFolder = `${req.body.domain}`
```

Comment:
You also need to add in the subdomain. You can create N number of repos per domain. not just 1 domain.